src/main/kotlin/com/chattriggers/ctjs/engine/IRegister.kt

```diff
@@ -522,68 +522,7 @@ interface IRegister {
         return EventTrigger(method, TriggerType.GuiKey, getImplementationLoader())
     }
 
-    /**
-     * Registers a new trigger that runs whenever the mouse is clicked with a
-     * gui open
-     *
-     * Passes through five arguments:
-     * - The mouse x position
-     * - The mouse y position
-     * - The mouse button
-     * - The gui
-     * - The event, which can be cancelled
-     *
-     * Available modifications:
-     * - [Trigger.setPriority] Sets the priority
-     *
-     * @param method The method to call when the event is fired
-     * @return The trigger for additional modification
-     */
-    fun registerGuiMouseClick(method: Any): EventTrigger {
-        return EventTrigger(method, TriggerType.GuiMouseClick, getImplementationLoader())
-    }
-
-    /**
-     * Registers a new trigger that runs whenever a mouse button is released
-     * with a gui open
-     *
-     * Passes through five arguments:
-     * - The mouse x position
-     * - The mouse y position
-     * - The mouse button
-     * - The gui
-     * - The event, which can be cancelled
-     *
-     * Available modifications:
-     * - [Trigger.setPriority] Sets the priority
-     *
-     * @param method The method to call when the event is fired
-     * @return The trigger for additional modification
-     */
-    fun registerGuiMouseRelease(method: Any): EventTrigger {
-        return EventTrigger(method, TriggerType.GuiMouseRelease, getImplementationLoader())
-    }
-
-    /**
-     * Registers a new trigger that runs whenever a mouse button held and dragged
-     * with a gui open
-     *
-     * Passes through five arguments:
-     * - The mouse x position
-     * - The mouse y position
-     * - The mouse button
-     * - The gui
-     * - The event, which can be cancelled
-     *
-     * Available modifications:
-     * - [Trigger.setPriority] Sets the priority
-     *
-     * @param method The method to call when the event is fired
-     * @return The trigger for additional modification
-     */
-    fun registerGuiMouseDrag(method: Any): EventTrigger {
-        return EventTrigger(method, TriggerType.GuiMouseDrag, getImplementationLoader())
-    }
+    // TODO(breaking): Remove registerGuiMouse{drag,click,release} - use clicked & dragged triggers instead
```

Comment:
`guiMouseClick` has quite a few uses actually. It seems pretty easy to keep support for these, just add it into the `MouseListener` register block surrounded by a check for the current screen. Though if we do go for that, I would like to get rid of `guiMouseRelease` and make `guiMouseClick` work like `clicked` (where it passes in the button state).

Reply:
Is it fine to remove guiMouseDrag? I understand that click has its uses, but what about drag?